tools/workspace.bzl

```diff
@@ -29,8 +33,12 @@ def initialize_third_party():
     boost()
     eigen()
     fastrtps()
+    ffmpeg()
     gflags()
+    glew()
```

Comment:
GLEW is not used across the whole Apollo project. So we should disable it in `tools/workspace.bzl`.